.github/workflows/scripts/slack_gem_notifications/notifications_methods.rb

```diff
@@ -39,10 +39,38 @@ def gem_updated?(versions)
   Time.now.utc - Time.parse(versions[0]['created_at']) < 24 * 60 * 60
 end
 
+def github_diff(gem_name, newest, previous)
+  diff = HTTParty.get(interpolate_github_url(gem_name, newest, previous))
+
+  diff.success?
+end
+
 def send_bot(gem_name, versions)
-  abort "Expected exactly 2 version numbers in the 'versions' array" unless versions.size == 2
+  abort("Expected exactly 2 version numbers in the 'versions' array") unless versions.size == 2
+  path = ENV['SLACK_GEM_NOTIFICATIONS_WEBHOOK']
+  options = {headers: {'Content-Type' => 'application/json'},
+             body: {
+              text: bot_message(gem_name, versions)}.to_json}
+
+  HTTParty.post(path, options)
+end
+
+def interpolate_github_url(gem_name, newest, previous)
+  "https://github.com/#{gem_name}/#{gem_name}/compare/v#{previous}...v#{newest}"
+end
```

Comment:
In a future iteration, there is an attribute called `source_code_uri` returned on some of RubyGems's API methods. This would likely have the exact GitHub address to help make this more successful when the gem's organization/username is not the same as the gem's name, or in cases like Unicorn,  let us know when the gem isn't hosted on GitHub at all.

Reply:
That is great! I'll save your comment in our bot ticket so this can be revisited.